inference-engine/samples/benchmark_app/CMakeLists.txt

```diff
@@ -2,11 +2,46 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+set(TARGET_NAME "benchmark_app")
+
 file (GLOB SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 
-ie_add_sample(NAME benchmark_app
+ie_add_sample(NAME ${TARGET_NAME}
               SOURCES ${SRC}
               HEADERS ${HDR}
               DEPENDENCIES format_reader ie_samples_utils
               OPENCV_DEPENDENCIES core)
+
+find_package(OpenCL)
+
+find_path(OpenCL_HPP_INCLUDE_DIR
+    NAMES
+      CL/cl2.hpp OpenCL/cl2.hpp
+    HINTS
+      ${opencl_root_hints}
+      ENV "PROGRAMFILES(X86)"
+      ENV AMDAPPSDKROOT
+      ENV INTELOCLSDKROOT
+      ENV NVSDKCOMPUTE_ROOT
+      ENV CUDA_PATH
+      ENV ATISTREAMSDKROOT
+      ENV OCL_ROOT
+    PATH_SUFFIXES
+      include
+      OpenCL/common/inc
+      "AMD APP/include")
```

Comment:
why do we need it? I believe `find_package(OpenCL)` already can find headers, is not it?

Reply:
It finds C headers only, while we need C++ OCL API as well.